wp-content/themes/engage/assets/scss/components/_garrett.scss

```diff
@@ -6,198 +6,197 @@
 }
 
 body {
+  background-color: transparent !important;
```

Comment:
Do you have some kind of autoformat on save on your editor? If so, could you turn it off while working on these files? It makes it really hard to review what you actually changed, since it's counting each line with different spacing as a new change.

Reply:
Yeah I do, I will disable it sorry!